pkgs/top-level/lua-generated-packages.nix

```diff
@@ -0,0 +1,279 @@
+/* ${GENERATED_NIXFILE} is an auto-generated file -- DO NOT EDIT! */
+{ self, stdenv
+, fetchurl
+, toLuaModule
+, requiredLuaModules
+}:
+with self;
+rec {
+mediator_lua = buildLuaPackage rec {
+meta={
+homepage="http://olivinelabs.com/mediator_lua/";
+license=stdenv.lib.licenses.mit;
+description="Event handling through channels"; }
+;
+src= fetchurl {
```

Comment:
Can you indent with two spaces in your generator and format the code like we do in nixpkgs in general?

Reply:
I will try to see what i can do in lua but I would rather run some nix prettifier. Any experience with https://github.com/Gabriel439/nixfmt or other advice ?

Reply:
`nixfmt` is probably not production ready. I am not aware of any other tool. Usual people write their tools in the same language as their target language. Then the likelihood is high that people, who are using the generator can also contribute back. Otherwise you can use the alignment feature of the `printf` function in bash.